### PR TITLE
WIP: update Japanese translation (Line 644-680)

### PR DESCRIPTION
**Pull request by yosshy**
This is work-in-progress for reviews by Japanese developers.
/assign @shu-mutou
/cc @atoato88 @YuikoTakada @t-inu

### CODE REVIEW

File: translations/ja.json [hunk 1]
```diff
@@ -641,43 +641,43 @@
...
-	"The path on the file system where the docs in markdown need to be saved": "",
-	"The path on the file system where the error code docs in markdown need to be saved": "",
-	"The path on the file system where the testing docs in markdown need to be saved": "",
-	"The podman service within '{{.cluster}}' is not active": "",
-	"The podman-env command is incompatible with multi-node clusters. Use the 'registry' add-on: https://minikube.sigs.k8s.io/docs/handbook/registry/": "",
-	"The requested memory allocation of {{.requested}}MiB does not leave room for system overhead (total system memory: {{.system_limit}}MiB). You may face stability issues.": "",
+	"The docker-env command is incompatible with multi-node clusters. Use the 'registry' add-on: https://minikube.sigs.k8s.io/docs/handbook/registry/": "docker-env コマンドはマルチノードクラスターと互換性がありません。'registry' アドオンを使用してください: https://minikube.sigs.k8s.io/docs/handbook/registry/",
+	"The docker-env command is only compatible with the \"docker\" runtime, but this cluster was configured to use the \"{{.runtime}}\" runtime.": "docker-env コマンドは \"docker\" ランタイムと互換性がありませんが、このクラスターは \"{{.runtime}}\" ランタイムを使用するよう設定されています。",
```
Comment (atoato88) [inline translations/ja.json:645]:
```suggestion
	"The docker-env command is only compatible with the \"docker\" runtime, but this cluster was configured to use the \"{{.runtime}}\" runtime.": "docker-env コマンドは \"docker\" ランタイムだけで互換性がありますが、このクラスターは \"{{.runtime}}\" ランタイムを使用するよう設定されています。",
```

File: translations/ja.json [hunk 2]
```diff
@@ -641,43 +641,43 @@
...
+	"The initial time interval for each check that wait performs in seconds": "実行待機チェックの最初の時間間隔 (秒)",
+	"The kubeadm binary within the Docker container is not executable": "Docker コンテナー内の kubeadm バイナリーが実行可能形式ではありません",
+	"The kubernetes version that the minikube VM will use (ex: v1.2.3)": "minikube VM で使用する Kubernetes バージョン（例: v1.2.3）",
+	"The machine-driver specified is failing to start. Try running 'docker-machine-driver-\u003ctype\u003e version'": "指定された machine-driver は起動に失敗しました。'docker-machine-driver-\u003ctype\u003e version' を実行してみてください",
+	"The minikube VM is offline. Please run 'minikube start' to start it again.": "minikube VM がオフラインです。'minikube start' を実行して minikube VM を再起動してください。",
+	"The minikube {{.driver_name}} container exited unexpectedly.": "minikube {{.driver_name}} コンテナーは想定外で終了しました。",
+	"The minimum required version for podman is \"{{.minVersion}}\". your version is \"{{.currentVersion}}\". minikube might not work. use at your own risk. To install latest version please see https://podman.io/getting-started/installation.html": "minikube が要求する podman のバージョンは \"{{.minVersion}}\" です。あなたのバージョンは \"{{.currentVersion}}\" です。minikube は動作しないかも知れません。自己責任で使用してください。",
```
Comment (atoato88) [inline translations/ja.json:659]: You missed translations for the sentence following: `To install latest version please see https://podman.io/getting-started/installation.html`

File: translations/ja.json [hunk 3]
```diff
@@ -641,43 +641,43 @@
...
-	"The requested memory allocation of {{.requested}}MiB does not leave room for system overhead (total system memory: {{.system_limit}}MiB). You may face stability issues.": "",
+	"The docker-env command is incompatible with multi-node clusters. Use the 'registry' add-on: https://minikube.sigs.k8s.io/docs/handbook/registry/": "docker-env コマンドはマルチノードクラスターと互換性がありません。'registry' アドオンを使用してください: https://minikube.sigs.k8s.io/docs/handbook/registry/",
+	"The docker-env command is only compatible with the \"docker\" runtime, but this cluster was configured to use the \"{{.runtime}}\" runtime.": "docker-env コマンドは \"docker\" ランタイムと互換性がありませんが、このクラスターは \"{{.runtime}}\" ランタイムを使用するよう設定されています。",
+	"The driver '{{.driver}}' is not supported on {{.os}}/{{.arch}}": "'{{.driver}}' ドライバーは {{.os}}/{{.arch}} に対応していません",
+	"The existing \"{{.name}}\" cluster was created using the \"{{.old}}\" driver, which is incompatible with requested \"{{.new}}\" driver.": "既存の \"{{.name}}\" クラスターは、(要求された \"{{.new}}\" ドライバーとは互換性のない) \"{{.old}}\" ドライバーを使用して作成されました。 ",
+	"The existing node configuration appears to be corrupt. Run 'minikube delete'": "既存のノード設定は破損している模様です。'minikube delete' を実行してください",
```
Comment (t-inu) [inline translations/ja.json:648]:
```suggestion
	"The existing node configuration appears to be corrupt. Run 'minikube delete'": "既存のノード設が破損しているようです。'minikube delete' を実行してください",
```

File: translations/ja.json [hunk 4]
```diff
@@ -641,43 +641,43 @@
...
+	"The heapster addon is depreciated. please try to disable metrics-server instead": "heapster アドオンは廃止予定です。代わりに metrics-server を無効化してみてください",
+	"The hyperv virtual switch name. Defaults to first found. (hyperv driver only)": "hyperv 仮想スイッチ名。デフォルト値は最初に見つかったスイッチ名です（hyperv ドライバーのみ）",
```
Comment (t-inu) [inline translations/ja.json:650]:
```suggestion
	"The hyperv virtual switch name. Defaults to first found. (hyperv driver only)": "hyperv 仮想スイッチ名。デフォルト値は最初に見つかったスイッチ名です。(hyperv ドライバーのみ)",
```

File: translations/ja.json [hunk 5]
```diff
@@ -641,43 +641,43 @@
...
+	"The hypervisor does not appear to be configured properly. Run 'minikube start --alsologtostderr -v=1' and inspect the error code": "ハイパーバイザーは適切に設定されていない模様です。'minikube start --alsologtostderr -v=1' を実行してエラーコードを確認してください",
```
Comment (t-inu) [inline translations/ja.json:651]:
```suggestion
	"The hypervisor does not appear to be configured properly. Run 'minikube start --alsologtostderr -v=1' and inspect the error code": "ハイパーバイザーが適切に設定されていないようです。'minikube start --alsologtostderr -v=1' を実行してエラーコードを確認してください",
```

File: translations/ja.json [hunk 6]
```diff
@@ -641,43 +641,43 @@
...
+	"The image '{{.imageName}}' was not found; unable to add it to cache.": "'{{.imageName}}' イメージは見つかりませんでした (キャッシュに追加できません)。",
+	"The initial time interval for each check that wait performs in seconds": "実行待機チェックの最初の時間間隔 (秒)",
```
Comment (t-inu) [inline translations/ja.json:653]:
```suggestion
	"The initial time interval for each check that wait performs in seconds": "実行待機チェックの初期時間間隔 (秒)",
```

File: translations/ja.json [hunk 7]
```diff
@@ -641,43 +641,43 @@
...
+	"The kubeadm binary within the Docker container is not executable": "Docker コンテナー内の kubeadm バイナリーが実行可能形式ではありません",
+	"The kubernetes version that the minikube VM will use (ex: v1.2.3)": "minikube VM で使用する Kubernetes バージョン（例: v1.2.3）",
```
Comment (t-inu) [inline translations/ja.json:655]:
```suggestion
	"The kubernetes version that the minikube VM will use (ex: v1.2.3)": "minikube VM で使用する Kubernetes バージョン (例: v1.2.3)",
```

File: translations/ja.json (hunk 1 above)
Comment (yosshy) [reply translations/ja.json:645]: done.

File: translations/ja.json (hunk 2 above)
Comment (yosshy) [reply translations/ja.json:659]: done

File: translations/ja.json (hunk 3 above)
Comment (yosshy) [reply translations/ja.json:648]: Applied.

File: translations/ja.json (hunk 4 above)
Comment (yosshy) [reply translations/ja.json:650]: done.

File: translations/ja.json (hunk 5 above)
Comment (yosshy) [reply translations/ja.json:651]: done.

File: translations/ja.json (hunk 7 above)
Comment (yosshy) [reply translations/ja.json:655]: done.

File: translations/ja.json (hunk 6 above)
Comment (yosshy) [reply translations/ja.json:653]: done.

**Comment by atoato88** [comment]
Thank you to update.
LGTM, I have no more comments.

**Comment by t-inu** [comment]
LGTM

**Comment by YuikoTakada** [comment]
Thank you for updating. Looks good to me. You just need to do rebase.

**Comment by atoato88** [comment]
@yosshy 
It looks some commits which does not related to this PR are includes.
Could you check again?

**Comment by yosshy** [comment]
> @yosshy It looks some commits which does not related to this PR are includes. Could you check again?

Sorry, looks my mistake. I'll find the cause and fix it.

**Comment by atoato88** [comment]
/lgtm
Thank you to update and sorry for late response.

**Pull request closed without merging**